### ISSUE TITLE
Add SSL

### ISSUE THREAD

**Issue by flynnduism** [deislabs/magick8sball.io#8]
I've been looking at this [caddyserver-autossl](https://github.com/yangl900/azure-aci-samples/tree/master/caddyserver-autossl) guide, but not sure if this is would suit the current config?

**Comment by carolynvs** [comment]
Yes, we just storage to make it work. Either something that is truly persistent, or just persistent enough like emptyDir.
